test/apicoverage/image/apicoverage-webhook.yaml

```diff
@@ -16,7 +16,7 @@ apiVersion: v1
 kind: Service
 metadata:
   name: apicoverage-webhook
-  namespace: knative-serving
+  namespace: {KNATIVE_SYSTEM_NAMESPACE}
```

Comment:
We use `knative-serving`...

I think what you are hitting is the absence of: https://github.com/knative/serving/blob/master/test/system.go

Reply:
system.go has changed the NS to `knative-testing` in this PR.

Reply:
This PR change the ns from default `knative-serving` into `knative-testing`.

Reply:
Hmm, I think I'd do this fairly differently.  I think that we should have a sane way of testing against existing installs and without running scripts to do it.

What I'd recommend would be:
 - Leave knative-serving as-is in our yamls,
 - Make `system.go` expose a flag that defaults to knative-serving,
 - Have the e2e test scripts generate a pseudo-random namespace and `s/knative-serving/<that>/`, and pass the namespace via the `system.go` flag.

Reply:
One cool / notable thing about this approach is that for systems that move Knative to a different namespace, they can run the e2e tests with `--systemNamespace=<where they put stuff>`.

Reply:
got it. thx.